taskcluster/tc-dotnet-utils.sh

```diff
@@ -34,7 +34,7 @@ install_nuget()
 
   ls -halR "${PROJECT_NAME}.${DS_VERSION}"
 
-  nuget install NAudio
+  nuget install NAudio -Version 1.10.0
```

Comment:
Hm. I wonder if this dependency and version spec should be in some project file to be installed automatically. Seems weird to have it just laying here. Maybe add a TODO comment? Right now is not the time to chase this I guess.

Reply:
Yeah, I agree and I'm unsure as well, so I'll keep #3540 open after we fix this and wait for carl's feedback on that point.